src/ColorUtils.php

```diff
@@ -2,33 +2,134 @@
 
 namespace CommandString\Utils;
 
-class ColorUtils {
-    public static function RGBAtoHEX(int $red, int $blue, int $green, ?int $alpha = null) {
-        $hex = [dechex($red), dechex($blue), dechex($green)];
-
-        if (!is_null($alpha)) {
-            $hex[] = dechex($alpha);
-        }
+/**
+ * ColourUtility is a PHP class that provides a set of utility functions for working with colours.
+ * 
+ * @method static array hexToRgb(string $hex)
+ * @method static string rgbToHex(int $r, int $g, int $b, $withPrefix = true)
+ * @method static int getBrightness(string $colour)
+ * @method static bool isDark(string $colour)
+ * @method static string getContrastColour(string $colour)
+ * @method static string getComplementaryColour(string $colour)
+ */
+class ColourUtility {
```

Comment:
```suggestion
class ColorUtils {
```
Do not rename the class, this needs to backwards compatible